src/devices/acurite.c

```diff
@@ -595,8 +597,10 @@ static int acurite_986_callback(bitbuffer_t *bitbuf) {
 	}
 
 	if ((status & 1) == 1) {
+            if (debug_output) {//All fprintf()'s must be under debug output guards.
 	    fprintf(stderr, "%s Acurite 986 sensor 0x%04x - %d%c: low battery, status %02x\n",
 		    time_str, sensor_id, sensor_num, sensor_type, status);
+            }
```

Comment:
I have a plan on how to go forward towards a release. This is part of it but it will take some time.